business/MetadataDatabase/Services/UserServices.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using MetadataDatabase.Repository;
+using MetadataDatabase.Data;
+using MetadataDatabase.Convertor;
+using MetadataDatabase.framework.DAL;
+using MongoDB.Bson;
+using MongoDB.Driver;
+
+namespace MetadataDatabase.Services
+{
+    public class UserServices : IUserServices
+    {
+        private readonly IUserRepository userRepository;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="UserServices"/> class.
+        /// </summary>
+        /// <param name="userRepository">The series repository.</param>
+        public UserServices(IUserRepository userRepository)
+        {
+            this.userRepository = userRepository;
+        }
+
+        /// <summary>
+        /// Creates the provided series.
+        /// </summary>
+        /// <param name="objectToCreate">The series to create.</param>
+        /// <returns></returns>
+        public UserDto Create(UserDto objectToCreate)
```

Comment:
Why not a userRegisterDto ?

Reply:
we are in the service, the HTTP POST & PUT request are the only entrypoint to create a user with password.

Due to the implementation of UserService based to UserDto type, it is not possible to change, it is that why we implemented setPassword method in order to manage password (may be we will update it with an encryption method)